### PR TITLE
adding the stub tests for global registration feature 6.10

### PR DESCRIPTION
**Pull request by omkarkhatavkar**

### CODE REVIEW

File: tests/foreman/ui/test_host.py [hunk 1]
```diff
@@ -1428,6 +1428,79 @@ def test_positive_global_registration_end_to_end(
...
 
 
+@pytest.mark.stubbed
+@pytest.mark.tier2
+def test_global_registration_form_populate(session):
+    """Host registration form should be get populate automatically based on the host-group
+
+    :id: b949e010-36b8-48b8-9907-36138342c72b
+
+    :expectedresults: Some of the fields in the form should be populated based on host-group
```
Comment (synkd) [inline tests/foreman/ui/test_host.py:1438]: Which values in the form will be populated based on the host group? Can you list them in `:expectedresults:`?

File: tests/foreman/ui/test_host.py [hunk 2]
```diff
@@ -1428,6 +1428,79 @@ def test_positive_global_registration_end_to_end(
         assert 'execution' in interface[0]['type']
 
 
+@pytest.mark.stubbed
+@pytest.mark.tier2
+def test_global_registration_form_populate(session):
+    """Host registration form should be get populate automatically based on the host-group
```
Comment (synkd) [inline tests/foreman/ui/test_host.py:1434]:
```suggestion
    """Host registration form should be populated automatically based on the host group
```

File: tests/foreman/ui/test_host.py [hunk 3]
```diff
@@ -1428,6 +1428,79 @@ def test_positive_global_registration_end_to_end(
...
+
+    :CaseLevel: Integration
+    """
+
+
+@pytest.mark.stubbed
+@pytest.mark.tier2
+def test_global_registration_token_restriction(session):
+    """Global registration token should be only used for registration call, it
+    should be restricted for the any other api calls.
```
Comment (synkd) [inline tests/foreman/ui/test_host.py:1493]:
```suggestion
    should be restricted for any other api calls.
```

File: tests/foreman/ui/test_host.py [hunk 4]
```diff
@@ -1428,6 +1428,79 @@ def test_positive_global_registration_end_to_end(
...
+
+    :id: 4528b5c6-0a6d-40cd-857a-68b76db2179b
+
+    :expectedresults: global registration token should be restricted any api calls
```
Comment (synkd) [inline tests/foreman/ui/test_host.py:1497]:
```suggestion
    :expectedresults: global registration token should be restricted for any api calls
```

**Review by synkd — CHANGES_REQUESTED** [review]

File: tests/foreman/ui/test_host.py [hunk 5]
```diff
@@ -1428,6 +1428,81 @@ def test_positive_global_registration_end_to_end(
...
+
+    :CaseLevel: Integration
+    """
+
+
+@pytest.mark.stubbed
+@pytest.mark.tier3
+def test_global_re_registration_host_with_force_ignore_error_options(session):
+    """If the ignore_error and force checkbox is checked then registered host can
+    get re-registered with out any error.
```
Comment (latran) [inline tests/foreman/ui/test_host.py:1481]:
```suggestion
    get re-registered without any error.
```

**Review by latran — CHANGES_REQUESTED** [review]

**Review by latran — COMMENTED** [review]
If this stays as manual in the future, I think adding test steps would help, but I'll leave that choice up to you.

**Review by devendra104 — APPROVED** [review]
ACK

**Review by JacobCallahan — APPROVED** [review]

**Pull request merged**